package.json

```
@@ -40,6 +42,13 @@
     "url": "https://github.com/particle-iot/firmware-protobuf/issues"
   },
   "homepage": "https://github.com/particle-iot/firmware-protobuf#readme",
+  "engines": {
+    "node": ">=16.x",
+    "npm": "8.x"
+  },
+  "files": [
+    "src"
```

I have a slight preference to include the proto files with the release as I find it useful to refer to those files from `node_modules/@particle/device-os-protobuf` rather than having to check out the device-os-proto repo to the right tag. If you're opposed, I don't mind leaving these out.
```suggestion
    "cloud",
    "control",
    "nanopb",
    "src"
```

---

I'm not opposed at all, it's indeed very useful.